src/Persistence/Migrations/20231022150158_EditedUserPasswordsEntityName.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Persistence.Migrations
+{
+    /// <inheritdoc />
+    public partial class EditedUserPasswordsEntityName : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropTable(
```

Comment:
This will not work as other developers who dont have that table  from experiment like you will cause an exception, the better solution is to remove the migration and add it again